packages/next-server/server/render.tsx

```diff
@@ -290,6 +295,10 @@ export async function renderToHTML(
       renderOpts.nextExport = true
     }
   }
+  // might want to change previewing of skeleton from `?skeleton=1`
+  isSkeleton =
+    pageConfig.experimentalPrerender === true &&
+    ((req as any).skipGIP || query.skeleton)
```

Comment:
`skeleton` as a query param is too generic IMO. I'd rather be verbose and name it `skipGetInitialProps=1`.

As this stands, it's an internal field only used for Next.js right? We really shouldn't be suggesting the users rely on it directly.

Reply:
The original idea was to allow previewing the skeleton. `skipGetInitialProps` wouldn't really be an accurate name since it calls it on the client-side just not during SSR. 

Right now it isn't relied on internally, if we changed to [this suggestion](https://github.com/zeit/next.js/pull/7983#discussion_r310260235) it would rely on it